test/rekt/features/channel/features.go

```diff
@@ -23,6 +23,7 @@ import (
 	cloudevents "github.com/cloudevents/sdk-go/v2"
 	"github.com/cloudevents/sdk-go/v2/binding"
 	"github.com/cloudevents/sdk-go/v2/test"
+	testlib "knative.dev/eventing/test/lib"
```

Comment:
it seems that this library has side effects when imported it defines cmd line flags.